src/testing.ts

```diff
@@ -96,6 +96,12 @@ const expectedResults = new Map<string, ResultType>([
   ['example_contracts/typeConversion/shifts', 'Success'],
   ['example_contracts/typeMinMax', 'Success'],
   ['example_contracts/units', 'Success'],
+  ['example_contracts/using_for/imports/global_directive', 'NotSupportedYet'],
+  ['example_contracts/using_for/imports/user_defined', 'NotSupportedYet'],
+  ['example_contracts/using_for/function', 'NotSupportedYet'],
+  ['example_contracts/using_for/private', 'TranspilationFailed'],
```

Comment:
We shouldn't be expecting tests to produce a TranspilationFailed, as that is the error for when something has gone wrong unexpectedly. I'd also appreciate it if you could add comments above entries marked NotSupportedYet or CairoCompileFailed to indicate what feature is blocking them

Reply:
During transpilation of examples_contract/using_for/private.sol, InheritanceInliner throws an error with msg `AssertionError [ERR_ASSERTION]: Attempted to copy non public/external function`.